README.md

```diff
@@ -78,7 +78,7 @@ like +NetworkPolicy+, +Ingress+ and +CustomMetrics+.
 
 ### Community Meeting
 
-* Community Meeting: every other Wednesday at 9:30-10:30 Pacific: [https://zoom.us/j/448032371](https://zoom.us/j/448032371)
+* Community Meeting: every other Wednesday at 10:00-10:30 Pacific: [https://zoom.us/j/448032371](https://zoom.us/j/448032371)
```

Comment:
My understand was that the meeting start time stays the same but it's 30m shorter

Reply:
From the March 18th [meeting minutes](https://docs.google.com/document/d/1NTBaJf6LhUBlF8_lfvBBt_MbyPvT-6CZNg6Ckpm_yCo/edit#):
```
- Adjust meeting time - could it start 30min later? (Michael)
  - We’re going to try just using the second half of this timeslot
```

Reply:
@mhausenblas ^^^

Reply:
Yup Stefan has it correct as far as I understood the consensus. I would, however, suggest to send out a mail to the list (by provisional chair?) to confirm and give everyone, especially folks who were not able to join, a chance to comment?

Reply:
I'm sorry, I meant to say that Lachie has it correct, starting at 10am PST ... I blame the new native GitHub client for the SNAFU 8-}